S16-io/eof.t

```diff
@@ -79,7 +79,7 @@ subtest '.eof on empty files' => {
         .read: 42;
         is-deeply .eof, True,  'eof is True after a read';
     }
-    with "/proc/$*PID/status".IO -> $p {
+    with "/proc/$*PID/environ".IO -> $p {
```

Comment:
I doubt the sanity of this test which is currently linux-like-centric. Neither `status` nor `environ` work for FreeBSD, for example. _/proc_ as such doesn't work neither on macos nor on Windows, etc. Guess, the file is to be chosen on platform-based grounds for better coverage.

- FreeBSD has `/proc/*/status`, as well as Linux.
- _environ_ would serve for Solaris then
- I'm not sure of the purpose of the test is to check for a special file accessibility, but macos has nothing but devfs mounted. So, _/etc/profile_ would be the best candidate after all
- Neither special mounts exists on Windows. So, _/Windows/win.ini_ or _system.init_ perhaps?

Reply:
All right, so how about making the /proc/*/environ solaris specific only?

Reply:
I'm only for it. And skip on macos and win*.